manifests/repo.pp

```diff
@@ -10,7 +10,7 @@
 			  include_src       => false,
 			}
 			
-			apt::key { 'hipchat':
+			apt::key { '0x69F57C04EA38EEE7A47E9BCCAAD4AA21729B5780':
```

Comment:
@nerdyness I have also bumped into this.

FWIW, I think this change would be more readable going forward if the key was put into the `id` parameter and the resource title was left as `hipchat`.

Example:

    apt::key { 'hipchat':
      id         => '0x69F57C04EA38EEE7A47E9BCCAAD4AA21729B5780',
      source => 'https://www.hipchat.com/keys/hipchat-linux.key',
    }